doc/source/tutorial/optimize.rst

```diff
@@ -337,11 +337,73 @@ Hessian product example:
     >>> res.x
     array([1., 1., 1., 1., 1.])
 
+Trust-Region Truncated Generalized Lanczos / Conjugate Gradient Algorithm (``method='trust-krylov'``)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Similar to the ``trust-ncg`` method, the ``trust-krylov`` method is a method
+suitable for large-scale problems as it uses the hessian only as linear
+operator by means of matrix-vector products.
+In contrast to the ``trust-ncg`` method, it solves the quadratic subproblem
```

Comment:
When you say "In contrast to the ``trust-ncg`` method, it solves the quadratic subproblem..." it is implying that the  ``trust-ncg`` method doesn't solve the quadratic subproblem. While it doesn't solve it exactly, it provides an approximate solution to it. So I think it would be better to rewrite this phrase. I would go for something like:
"it solves the quadratic subproblem more accurately than the ``trust-ncg`` method..."